sphinx_immaterial/theme.conf

```
@@ -11,32 +10,42 @@ direction =
 
 icon =
 
-# TODO: This could be improved
-# Path to a touch icon. Should be 152x152 or larger. Path is relative
-# to _static, so that if your icon is in _static/images/touch-icon.png,
-# touch_icon would be images/touch-icon.png. Icon must be in _static since
-# this file is not copied
-touch_icon =
-
 font =
 
+# Some features available from original mkdocs theme
+# navigation.expand = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion
+# navigation.tabs = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs
+# toc.integrate = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-integration
+# navigation.sections = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections
+# navigation.instant = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-loading
+# header.autohide = https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#automatic-hiding
+# navigation.top = https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#back-to-top-button
+# search.highlight = https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#search-highlighting
+# search.share = https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#search-sharing
 features =
 
+# See original mkdocs-material site for in-depth configuration
+# https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#configuration
 palette =
 
 # Set to an array `["UA-XXXXX", "auto"]` to enable google analytics
 google_analytics =
 
 # Set to disqus id to enable disqus integration
+# See https://squidfunk.github.io/mkdocs-material/setup/adding-a-comment-system/#disqus
 disqus =
 
 # Repository integration
+# See https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#configuration
 # Set the repo url for the link to appear
 repo_url =
 # The name of the repo. If must be set if repo_url is set
 repo_name =
 # Must be one of github, gitlab or bitbucket
 repo_type = github
+# url segment that is concatenated with repo_url to point readers to the document's
+# source file. This is typically in the form of 'blob/<branch name>/<docs source folder>'.
+edit_uri = blob/main/docs
```

Weren't you going to leave this blank by default?

---

Whoops. Testing artifact.
```suggestion
edit_uri =
```